examples/geopm_platform_supported.cpp

```diff
@@ -3,62 +3,61 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-#include <iostream>
-#include <iomanip>
+#include <fcntl.h>
 #include <string.h>
 #include <unistd.h>
-#include <sys/types.h>
+#include <iomanip>
+#include <iostream>
 #include <sys/stat.h>
-#include <fcntl.h>
+#include <sys/types.h>
 
 #include "geopm/PlatformTopo.hpp"
-#include "geopm_hash.h"
 #include "geopm_error.h"
-
+#include "geopm_hash.h"
 
 int main(int argc, char **argv)
 {
     int err = 0;
     char error_msg[512];
     error_msg[511] = '\0';
 
-    if (argc == 2 &&
-        strncmp(argv[1], "crc32", strlen("crc32") + 1) == 0) {
+    if (argc == 2 && strncmp(argv[1], "crc32", strlen("crc32") + 1) == 0) {
         uint64_t result = geopm_crc32_u64(0xDEADBEEF, 0xBADFEE);
-        if (result == 0xA347ADE3 ) {
+        if (result == 0xA347ADE3) {
             std::cout << "Platform supports crc32 intrinsic." << std::endl;
         }
         else {
             err = GEOPM_ERROR_PLATFORM_UNSUPPORTED;
-            std::cerr << "Warning: <geopm_platform_supported>: Platform does not support crc32 intrinsic." << std::endl;
+            std::cerr << "Warning: <geopm_platform_supported>: Platform does not support crc32 intrinsic."
+                      << std::endl;
         }
     }
     else {
         int cpu_id = geopm_read_cpuid();
-        if (cpu_id != 0x63F &&
-            cpu_id != 0x63F &&
-            cpu_id != 0x63E &&
-            cpu_id != 0x62D &&
-            cpu_id != 0x64F &&
-            cpu_id != 0x657) {
+        if (cpu_id != 0x63F && cpu_id != 0x63F && cpu_id != 0x63E &&
+            cpu_id != 0x62D && cpu_id != 0x64F && cpu_id != 0x657) {
```

Comment:
I liked it the old way personally.  it made maintenance easy.

Reply:
Maybe related this and other line-breaking-related comments:

Setting `ColumnLimit` to 0 will tell `clang-format` to "respect the input’s line breaking decisions within statements unless they contradict other rules." Otherwise, messing with the weights of some of the line-breaking penalties might help get closer to the desired behavior. The penalties allow us to prioritize how a line is wrapped when wrapping is needed.

Maybe that'll resolve some of these cases.

Reply:
I'm not sure I'm willing to do away with the column limit feature of clang for this particular use case.  The output looks okay to me.